### ISSUE TITLE
TextArea widget in Flutter

### ISSUE THREAD

**Issue by programming6131** [flutter/flutter#62397]
I want to suggest implementation of a new Input widget that is TextArea widget ,
rather than using TextField and adding the properties maxLines and maxLength to create a feel of TextArea

**Comment by Astrallis** [comment]
Yes, we seriously need this 😅😅

**Comment by HansMuller** [comment]
CC @justinmc

**Comment by justinmc** [comment]
Would it have any new behavior that's not possible now with TextField, or would it just be a special case of TextField that has different default values for maxLines and maxLength?

**Comment by Astrallis** [comment]
It can have some stuff, like color, some decoration properties like that of a container. Max characters, disabling the enabledBorder, disabledBorder, focusedBorder by default. 
What I mean to say is, that instead of wrapping a textfield in a container to give it a presentable look, these things can be provided by default.

**Comment by programming6131** [comment]
> 
> 
> It can have some stuff, like color, some decoration properties like that of a container. Max characters, disabling the enabledBorder, disabledBorder, focusedBorder by default.
> What I mean to say is, that instead of wrapping a textfield in a container to give it a presentable look, these things can be provided by default.

I could'nt  agree more, this unique features to be present in the widget by default and any other property you can think of that suites a TextArea

**Comment by programming6131** [comment]
They say in flutter everything is a widget and I feel this proposed widget should be implemented because any widget i think of during development should be there for me

**Comment by programming6131** [comment]
> 
> 
> Would it have any new behavior that's not possible now with TextField, or would it just be a special case of TextField that has different default values for maxLines and maxLength?

yes it has it should have  its own special properties like a border, color and some other properties borrowed from existing input widgets

**Comment by justinmc** [comment]
I think this would make a great Pub package project if one doesn't already exist.

Otherwise I'll keep an eye on this issue and see how much interest there is for something like this in Flutter itself.

**Comment by icecandy** [comment]
+1

**Comment by Seraj2335** [comment]
Yes I need this
